zio-http/src/main/scala/zhttp/service/Handler.scala

```diff
@@ -24,17 +20,22 @@ private[zhttp] final case class Handler[R](
     with WebSocketUpgrade[R] { self =>
 
   override def channelRead0(ctx: ChannelHandlerContext, jReq: FullHttpRequest): Unit = {
-    jReq.touch("server.Handler-channelRead0")
     implicit val iCtx: ChannelHandlerContext = ctx
+    val headers                              = Headers.make(jReq.headers())
+    val content                              = Unpooled.copiedBuffer(jReq.content())
```

Comment:
Why is this required? It's going to affect performance.

ALSO — If heap is the way to go for us, then we can set that configuration one time when the server starts I think.